components/layout/Search.vue

```diff
@@ -65,14 +137,33 @@
                             </div>
                         </div>
                     </div>
-                    <div class="facets">
-                        <div class="facet" :class="{'facet-active': selectedFacet === undefined}" @click="() => selectFacet(undefined)">
+                    <div class="facets overflow-x-auto overflow-y-hidden">
+                        <div
+                            class="facet"
+                            :class="{
+                                'facet-active': selectedFacet === undefined,
+                            }"
+                            @click="() => selectFacet(undefined)"
+                        >
                             <span>All</span>
-                            <span class="badge rounded-pill bg-primary">{{ allSum }}</span>
+                            <span class="badge rounded-pill bg-primary">{{
+                                allSum
+                            }}</span>
                         </div>
-                        <div class="facet" v-for="(result, key, index) in searchFacets" @click="() => selectFacet(key)" :class="{'facet-active': selectedFacet === key}" :key="index">
-                            <span>{{key.charAt(0).toUpperCase() + key.slice(1).toLowerCase()}}</span>
-                            <span class="badge rounded-pill bg-primary">{{ result }}</span>
+                        <div
+                            class="facet"
+                            v-for="(result, key, index) in searchFacets"
+                            @click="() => selectFacet(key)"
+                            :class="{ 'facet-active': selectedFacet === key }"
+                            :key="index"
+                        >
+                            <span>{{
+                                key.charAt(0).toUpperCase() +
+                                key.slice(1).toLowerCase()
```

Comment:
I find the old inline style easier to read (simple string transform + concat like that can be one-lined imo)